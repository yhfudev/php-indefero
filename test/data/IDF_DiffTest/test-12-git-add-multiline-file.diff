diff --git a/b b/b
new file mode 100644
index 0000000..bfdaa0f
--- /dev/null
+++ b/b
@@ -0,0 +1,3 @@
+a
+b
+
