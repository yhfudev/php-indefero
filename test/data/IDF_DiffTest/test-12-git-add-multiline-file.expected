<?php return array (
  'b' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '',
          1 => '1',
          2 => "a\n",
        ),
        1 => 
        array (
          0 => '',
          1 => 2,
          2 => "b\n",
        ),
        2 => 
        array (
          0 => '',
          1 => 3,
          2 => "\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '0',
          1 => '0',
        ),
        1 => 
        array (
          0 => '1',
          1 => '3',
        ),
      ),
    ),
  ),
);